Refresh.GameServer/Database/GameDatabaseContext.Photos.cs

```diff
@@ -106,4 +107,21 @@ public int GetTotalPhotosWithUser(GameUser user)
     [Pure]
     public int GetTotalPhotosInLevel(GameLevel level)
         => this.GamePhotos.Count(p => p.LevelId == level.LevelId);
+    
+    public void DeletePhotosPostedByUser(GameUser user)
+    {
+        IEnumerable<GamePhoto> photos = this.GamePhotos.Where(s => s.Publisher == user);
+        
+        this.Write(() =>
+        {
+            foreach (GamePhoto photo in photos)
+            {
+                IQueryable<Event> photoEvents = this.Events
+                    .Where(e => e._StoredDataType == (int)EventDataType.Photo && e.StoredSequentialId == photo.PhotoId);
+                
+                this.Events.RemoveRange(photoEvents);
+                this.GamePhotos.Remove(photo);
```

Comment:
You can also just directly use `DeletePhoto` here; since with the newer Realm logic we have we can chain have nested write queries.

Should probably also move the event removing code there as well.